packages/grafana-runtime/src/internal/index.ts

```diff
@@ -27,3 +27,5 @@ export {
 } from '../services/pluginExtensions/getObservablePluginLinks';
 
 export { UserStorage } from '../utils/userStorage';
+
+export { initOpenFeature, getOpenFeatureClient, evaluateBooleanFlag } from './openFeature';
```

Comment:
Silly question, but do we not expect any of the cloud plugins to use openFeature?

Reply:
I did write this down somewhere but I can’t remember where… but at the moment the approach is for cloud plugins to create their own OpenFeature clients

Reply:
I see, thanks for clarifying. 

>I’m happy to pause this if there’s work you would like to do for the implementation for all plugins.

I’m not sure about the backend side, but I’m not aware of any frontend work that would require you to pause. We will eventually need new APIs for e2e testing with feature toggles, but there’s no urgency since this is just a basic initial implementation.